### PR TITLE
Ion channel families imported from HGNC

### PR DESCRIPTION
**Pull request by bgyori**
This PR adds an automated script to generate FamPlex entities, relations, and equivalences from HGNC families. It also applies this specifically to groups of ion channels curated in HGNC, and imports them into FamPlex. Redundancies between HGNC groups and FamPlex entries are found and resolved. This PR doesn't yet add grounding map entries for these families, this will be done through curation.

### CODE REVIEW

**Comment by bgyori** [comment]
Another note is that all the relations used in this import are `isa`. In some cases, a `partof` relation may be warranted, but this would have to be done manually by reading up on each entry.

File: equivalences.csv [hunk 1]
```diff
@@ -1582,3 +1582,50 @@ SIGNOR,SIGNOR-PF6,AMPK_alpha
 SIGNOR,SIGNOR-PF7,p14_3_3
 SIGNOR,SIGNOR-PF8,FGF
 SIGNOR,SIGNOR-PF9,BMP_receptor
+HGNC_GROUP,161,Ligand_gated_ion_channels
```
Comment (cthoyt) [inline equivalences.csv]: Why not use HGNC_GENEFAMILY to better mirror identifiers.org?

File: relations.csv [hunk 2]
```diff
@@ -2800,3 +2791,349 @@ HGNC,NR1H2,isa,FPLX,LXR
...
+HGNC,TRPV1,isa,FPLX,TRP
+HGNC,TRPV2,isa,FPLX,TRP
+HGNC,TRPV3,isa,FPLX,TRP
+HGNC,TRPV4,isa,FPLX,TRP
+HGNC,TRPV5,isa,FPLX,TRP
+HGNC,TRPV6,isa,FPLX,TRP
+HGNC,VDAC1,isa,FPLX,VDAC
+HGNC,VDAC2,isa,FPLX,VDAC
+HGNC,VDAC3,isa,FPLX,VDAC
+FPLX,CACN,isa,FPLX,Voltage_gated_ion_channels
```
Comment (cthoyt) [inline relations.csv]: Great that this is capturing the hierarchy already curated by HGNC!

File: grounding_map.csv [hunk 3]
```diff
@@ -2659,3 +2659,4 @@ Rp1,HGNC,RP1,,,,
 RP-1,HGNC,RP1,,,,
 Psk2,UP,Q08217,,,,
 PSK2,UP,Q08217,,,,
+liposomes,MESH,D008081,,,,
```
Comment (cthoyt) [inline grounding_map.csv]: I don't think this is relevant for the PR... just pointing it out to make sure it doesn't get lost or forgotten :)

**Comment by cthoyt** [comment]
I had a couple of comments which you might want to consider, but do you have a plan/todo list for when you're going to merge this PR?

File: equivalences.csv (hunk 1 above)
Comment (bgyori) [reply equivalences.csv]: I just followed the existing entries, I'm not against renaming them all together, just not on this PR necessarily

File: grounding_map.csv (hunk 3 above)
Comment (bgyori) [reply grounding_map.csv]: Yes, just a local commit that ended up on the PR, but I was going to push it anyway so let's leave it

**Comment by bgyori** [comment]
I think this PR is basically ready to go, though I have a couple of minor things to change (some equivalences and ordering some files), and @johnbachman hasn't commented on it yet

**Pull request merged**